MNE/rtProcessing/rthpis.cpp

```diff
@@ -79,113 +79,93 @@ using namespace INVERSELIB;
 // DEFINE MEMBER METHODS
```

Comment:
Since there are residing two classes in this file consider having a more explain comment here:

============================================================================================================= 
 // DEFINE MEMBER METHODS OF RtHPISWorker
 //=============================================================================================================